src/ResourceManager/DataLakeStore/Commands.DataLakeStore/help/Get-AzureRmDataLakeStoreChildItemSummary.md

```diff
@@ -0,0 +1,142 @@
+---
+external help file: Microsoft.Azure.Commands.DataLakeStore.dll-Help.xml
+Module Name: AzureRM.DataLakeStore
+online version:
+schema: 2.0.0
+---
+
+# Get-AzureRmDataLakeStoreChildItemSummary
+
+## SYNOPSIS
+Gets the summary of total size, files and directories contained in the path specified
+
+## SYNTAX
+
+```
+Get-AzureRmDataLakeStoreChildItemSummary [-Account] <String> [-Path] <DataLakeStorePathInstance>
+ [-Concurrency <Int32>] [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+{{Fill in the Description}}
```

Comment:
@rahuldutta90 please add a description for this cmdlet

Reply:
@cormacpayne Fixed